Client/Ecosia/Settings/EcosiaSettings.swift

```diff
@@ -176,11 +176,19 @@ final class EcosiaSendAnonymousUsageDataSetting: BoolSetting {
                   prefKey: "",
                   defaultValue: true,
                   titleText: .localized(.sendUsageDataSettingsTitle),
-                  statusText: .localized(.sendUsageDataSettingsDescription),
+                  statusText: .localized(.sendUsageDataSettingsDescription) + " " + .localized(.learnMore),
```

Comment:
It may be cleaner to read. It shouldn't have any side effects
```suggestion
                  statusText: "\(.localized(.sendUsageDataSettingsDescription)) \(.localized(.learnMore))",
```

Not sure whether it needs to escape `)`

Reply:
I tried it, but the problem is that the compiler doesn't assume it is String then, which means we would need something like:
```
"\(String.localized(.sendUsageDataSettingsDescription)) \(String.localized(.learnMore))"
```
I don't really find it readable then, that's why I opted for the appending approach, but let me know if you disagree 🙂